.circleci/config.yml

```diff
@@ -506,6 +548,14 @@ jobs:
                     -- \
                     <<#parameters.with_core>>-m 'not wandb_core_failure' <</parameters.with_core>> \
                     tests/pytest_tests/system_tests/test_notebooks
+      - when:
+          condition: { equal: [<<parameters.tests>>, importers] }
+          steps:
+            - run:
+                name: Run tests for <<parameters.tests>>
+                no_output_timeout: 10m
+                command: |
+
```

Comment:
you don't need this if you have the importers job (it's either using the system-tests or having your own job)

Reply:
sorry, dangling